consumer.go

```diff
@@ -16,10 +17,13 @@ var (
 // NewConsumer creates a new kinesis connection and returns a
 // new consumer initialized with app and stream name
 func NewConsumer(appName, streamName string) *Consumer {
-	sess := session.New(
-		aws.NewConfig().WithMaxRetries(10),
+	log.SetHandler(text.New(os.Stderr))
```

Comment:
could potentially wrap this with some type of conditional and not call `SetHandler` if it's already been set.